p2composite.example.parent/pom.xml

```diff
@@ -21,11 +21,9 @@
   </modules>
 
   <properties>
-    <tycho-version>2.2.0</tycho-version>
-    <tycho-extras-version>${tycho-version}</tycho-extras-version>
-    <maven-antrun-plugin.version>1.7</maven-antrun-plugin.version>
+    <tycho-version>4.0.2</tycho-version>
```

Comment:
Tycho 4 requires Java 17, so the GitHub Actions workflow should be updated accordingly (currently, it uses Java 11, in fact, the workflow fails).